# horizon
T_horizon: 150

# controls
init_control_seq: [0.165206,-0.038453,0.332158,-0.119533,0.441817,-0.093453,0.511277,-0.167540,0.569332,-0.148113,0.615540,-0.204716,0.657406,-0.142532,0.610913,-0.155587,0.609537,-0.171511,0.685762,-0.133032,0.796218,-0.162689,0.803878,-0.203420,0.835645,-0.187674,0.906718,-0.228658,0.968552,-0.247026,0.993902,-0.246061,1.000000,-0.238111,1.000000,-0.243191,1.000000,-0.213303,1.000000,-0.235469,1.000000,-0.266127,1.000000,-0.233154,1.000000,-0.205718,1.000000,-0.169753,1.000000,-0.102678,1.000000,-0.051436,1.000000,-0.060789,1.000000,-0.045827,1.000000,-0.052595,1.000000,-0.032274,1.000000,-0.109890,1.000000,-0.186914,0.994116,-0.116274,0.983064,-0.061713,1.000000,-0.063358,1.000000,-0.097012,1.000000,-0.099612,1.000000,-0.107307,1.000000,-0.073128,1.000000,-0.035470,0.999776,0.010548,0.992551,0.024362,0.985967,0.053112,0.993499,0.087344,1.000000,0.119186,0.996890,0.083417,0.995752,0.080148,0.998647,0.062572,1.000000,0.058646,1.000000,0.025695,0.997509,-0.014978,0.965250,-0.069572,0.995306,-0.032799,1.000000,0.049027,1.000000,0.087192,1.000000,0.164319,1.000000,0.259501,1.000000,0.347695,1.000000,0.397571,0.999340,0.405658,1.000000,0.478231,0.997745,0.402245,0.972614,0.425027,0.999751,0.449429,1.000000,0.428429,1.000000,0.436479,1.000000,0.405930,1.000000,0.398733,1.000000,0.367699,1.000000,0.362628,1.000000,0.272278,0.977712,0.276156,1.000000,0.239802,1.000000,0.183632,0.999879,0.170790,1.000000,0.152846,0.994621,0.071878,1.000000,0.031850,0.999962,0.009596,1.000000,0.020609,1.000000,0.006976,1.000000,-0.000321,1.000000,0.006310,1.000000,-0.002245,1.000000,0.006654,0.999999,-0.008458,0.986695,0.003414,0.994916,-0.012583,0.998045,-0.021600,1.000000,-0.070463,1.000000,-0.000888,0.988636,-0.000258,1.000000,0.008264,1.000000,-0.019407,1.000000,-0.065213,1.000000,-0.098317,1.000000,-0.067071,1.000000,-0.072346,1.000000,-0.058108,1.000000,-0.001648,1.000000,0.036308,1.000000,0.006677,1.000000,0.003019,1.000000,-0.018936,1.000000,-0.026375,1.000000,-0.036316,1.000000,-0.029307,1.000000,0.011302,1.000000,0.041159,0.999119,-0.006368,1.000000,0.002065,1.000000,0.056166,0.998007,0.087836,1.000000,0.047334,1.000000,0.042923,1.000000,0.019219,1.000000,0.014194,1.000000,0.000445,0.998494,-0.006319,1.000000,-0.016873,1.000000,-0.023840,1.000000,-0.034100,1.000000,-0.048321,1.000000,-0.064718,1.000000,-0.096304,0.996884,-0.194652,1.000000,-0.131458,1.000000,-0.113034,1.000000,-0.089057,1.000000,-0.053433,1.000000,-0.029712,0.952280,0.033216,0.999687,0.023927,1.000000,-0.033850,0.991897,-0.106234,1.000000,-0.079593,1.000000,-0.075319,0.999625,-0.025632,1.000000,0.034732,0.985806,0.092629,1.000000,0.020382,1.000000,-0.032760,1.000000,-0.061840,0.999897,-0.073309,0.998013,-0.117521,1.000000,-0.160210,0.999717,-0.156302,1.000000,-0.173568,0.999492,-0.171259,0.994060,-0.101973]
